internal/pkg/n8n-client-go/workflows.go

```diff
@@ -93,3 +93,26 @@ func (c *Client) DeleteWorkflow(workflowID string) (*Workflow, error) {
 
 	return &workflow, nil
 }
+
+// ActivateWorkflow - Activates a workflow by its ID.
+func (c *Client) ActivateWorkflow(workflowID string) (*Workflow, error) {
+	url := fmt.Sprintf("%s/api/v1/workflows/%s/activate", c.HostURL, workflowID)
+
+	req, err := http.NewRequest("POST", url, nil)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Not an issue but I noticed that `ActivateWorkflow` has some minor differences in comparison to the existing methods `GetWorkflow`, `DeleteWorkflow` and the new method `DeactivateWorkflow` introduced by #18 .

Shouldn't we maintain the same code pattern across all the unit tests for better code readability?

```suggestion
	req, err := http.NewRequest("POST", fmt.Sprintf("%s/api/v1/workflows/%s/activate", c.HostURL, workflowID), nil)
	if err != nil {
		return nil, err
	}
```

**Note**: if you prefer to use the new standard, maybe we should adjust the other methods?